docs/source/monitors.rst

```diff
@@ -89,8 +91,7 @@ Here is an example of how to configure a new monitor suite in your project:
 
   ``name`` suite name
 
-  ``monitors`` list of :class:`~spidermon.core.monitors.Monitor` that will be executed
-  if this suite is enabled.
+  ``monitors`` list of :class:`Monitor` that will be executed if this suite is enabled.
```

Comment:
After applying the change I mention above, the change in this line should be reverted, and it should work nonetheless.

Reply:
Tick